hermes/wasm/crates/wasi-component-adapter/byte-array-literals/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
```

Comment:
I think that `byte-array-literals` is actually compiled as it is used inside `wasi-component-adepter` and it has already been included in `hermes/wasm/Cargo.toml`. But the `verify` crate is surely not compiled as it is a binary target to check the compiled `.wasm` output. So it is unused right now.

Reply:
I have tried to compile `verify` crate and it did not compile.
Maybe lets include both of these crate explicitly as workspace members as it was done here https://github.com/bytecodealliance/wasmtime/blob/37300d3f4b51e0e3374e3c4fc382b7603b065c8b/Cargo.toml#L115

Reply:
I included the `verify` crate into workspace members and tried to compile it. To compile, you need to specify the build target to your local machine target since `wasm32-unknown-unknown` was set by default. But I found a problem while running the bin to verify the output wasm here.

```bash
Error: unsupported section DataSection("...") found in preview1.wasm
```

Reply:
Lets make the `verify` crate being built and working out of scope for this PR.  Leave the code in-tree but we will pick it up with a future set of work.